app.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const { getTopics } = require("./Controllers/controller.js");
+
+const app = express();
+
+app.use(express.json());
+app.get("/api/topics", getTopics);
+app.get("/api/articles",getArticles);
+
+// for next endpoint
+app.use((err, req, res, next) => {
+  if (err.msg && err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
```

Comment:
The above error handler looks good - but not yet necessary - I would remove for now and reintroduce when relevant later